src/lib.rs

```diff
@@ -18,12 +19,15 @@ extern crate libc;
 mod macros;
 
 mod common;
+pub mod i386;
 pub mod instruction_emulator;
+pub mod interrupts;
 mod platform;
 mod win_hv_emulation;
 mod win_hv_emulation_defs;
 mod win_hv_platform;
 mod win_hv_platform_defs;
+mod win_hv_platform_defs_internal;
```

Comment:
This is missing. Is this just a leftover or is it a file that was supposed to be included in the commit?